### DISCUSSION TITLE
Long-term data safety

### DISCUSSION THREAD

**Discussion by wujastyk** [PanditProject/Website#50] [Help]
I have recently been giving thought to backing up my own [research project](http://sushrutaproject.org), and it occurred to me that some of the same issues face PanditProject.  I have written a [short blog post](https://sushrutaproject.org/2022/04/20/long-term-data-security/) on the topic of long-term data safety and explaining what we're now doing at our project. 

Would [Zenodo](https://about.zenodo.org/) also be an appropriate place to back up PanditProject software and data?

Best,
Dominik

**Comment by ybronner** [comment]
Thanks for sharing this blog. I think this is good idea, and we should discuss this.
